src/rustllvm/rustllvm.h

```diff
@@ -45,7 +45,16 @@
 #include "llvm-c/ExecutionEngine.h"
 #include "llvm-c/Object.h"
 
-#if LLVM_VERSION_MINOR >= 7
+#define LLVM_VERSION_GT_OR_EQ(major, minor) \
+  LLVM_VERSION_MAJOR > (major) || LLVM_VERSION_MAJOR == (major) && LLVM_VERSION_MINOR >= (minor)
```

Comment:
Could this entire clause also be wrapped in parens? (yay C macros)

same below as well